### ISSUE TITLE
Nightly scripts aren't running. 

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/openFrameworks#8193]
https://github.com/openframeworks/openFrameworks/actions/runs/11761061978 

```
[Invalid workflow file: .github/workflows/nightly-linux64.yml#L48](https://github.com/openframeworks/openFrameworks/actions/runs/11761061978/workflow)
You have an error in your yaml syntax on line 48
```

<img width="972" alt="image" src="https://github.com/user-attachments/assets/da3a6e6c-4892-4c42-9a00-715f37bb50cb">

**Comment by dimitre** [comment]
https://github.com/openframeworks/openFrameworks/issues/8194

**Comment by dimitre** [comment]
I can fix that yml syntax

**Comment by ofTheo** [comment]
closed by #8195
